packages/lib-near/package.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "@dfns/lib-near",
+  "version": "0.7.5",
+  "peerDependencies": {
+    "near-api-js": "6.2.4"
```

make it a bit more flexible

```suggestion
    "near-api-js": "^6.0.0"
```